impl/shim/mapreduce/src/main/java/org/pentaho/big/data/impl/shim/mapreduce/MapReduceServiceImpl.java

```diff
@@ -89,7 +89,14 @@ public MapReduceJobSimple executeSimple( URL resolvedJarUrl, String driverClass,
 
     final List<String> finalClassNamesInJarWithMain = classNamesInJarWithMain;
 
-    final Class<?> mainClassFromManifest = jarUtility.getMainClassFromManifest( resolvedJarUrl, classLoader );
+    Class<?> mainClassFromManifest = null;
+    try {
+      mainClassFromManifest = jarUtility.getMainClassFromManifest( resolvedJarUrl, classLoader );
+    } catch ( Exception e ) {
```

Comment:
@mkambol I was trying to preserve the same flow [from before.](https://github.com/pentaho/big-data-plugin/blob/b46d4cd1d7681044d9f03f7a9a33d8bd55cc8e0c/legacy/src/main/java/org/pentaho/di/ui/job/entries/hadoopjobexecutor/JobEntryHadoopJobExecutorController.java#L566)  The user is able to type in a main class or select from autodetected ones even if there is no manifest, the file isn't there, etc.